invenio_config_tugraz/templates/security/email/welcome.txt

```diff
@@ -0,0 +1,17 @@
+{% trans salut=current_userprofile.full_name or user.email %}
+Dear {{ salut }}!
+{% endtrans %}
+{{ _('Welcome to TU Graz Repository!') }}
```

Comment:
thats not required - since the subject says exactly the same see here: https://github.com/tu-graz-library/invenio-config-tugraz/blob/60b85e6ad576de010cb54f3e17f6c3caf07b6d05/invenio_config_tugraz/config.py#L87
```suggestion

```